spec/jobs/search/remove_from_index_worker_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+require "jobs/shared_examples/enqueues_job"
+
+RSpec.describe Search::RemoveFromIndexWorker, type: :worker do
+  include_examples "#enqueues_job", "search_remove_from_index",
+                   "searchables_#{Rails.env}", "users-456"
```

Comment:
this can become:

```ruby
  include_examples "#enqueues_on_correct_queue", "default", # params
```

Reply:
I've changed that 👍 what the reasoning behind changing that one over? (Just for future reference)

Reply:
it's not a change, it's two different examples, `#enqueues_job` work with active jobs, `enqueues_on_correct_queue` works with sidekiq